AppCenterDataStorage/AppCenterDataStorageTests/MSDataStoreTests.m

```diff
@@ -617,12 +634,13 @@ - (void)testCreateWithPartitionWhenCreationFails {
                                                              tokenResult:testToken
                                                               documentId:kMSDocumentIdTest
                                                               httpMethod:kMSHttpMethodPost
-                                                                    body:OCMOCK_ANY
+                                                                document:mockSerializableDocument
                                                        additionalHeaders:OCMOCK_ANY
+                                                       additionalUrlPath:OCMOCK_ANY
                                                        completionHandler:OCMOCK_ANY])
       .andDo(^(NSInvocation *invocation) {
         MSHttpRequestCompletionHandler cosmosdbOperationCallback;
-        [invocation getArgument:&cosmosdbOperationCallback atIndex:8];
+        [invocation getArgument:&cosmosdbOperationCallback atIndex:9];
```

Comment:
If this PR goes after #1513, there should be one more place that requires this change.